src/App.vue

```diff
@@ -192,4 +312,23 @@
         max-height: 100%;
         overflow: hidden;
     }
+
+    .header-bg {
+      background: #f7f7f7;
+    }
+
+    .validation-panel {
+      background: #f7f7f7;
+      height: 10rem;
+      width: 21.35rem;
+      bottom: 3.5rem;
+      right: 0;
+    }
+    .dataInput {
+      margin-top: -25px;
+    }
+
+    .h-80 {
+      height: 80vh;
```

Comment:
This might conflict with the bootstrap util classes.

Reply:
We should try and eliminate custom CSS as much as possible. We should be using only bootstrap classes.